### ISSUE TITLE
Add job priorities that define the order on which jobs are run on each execution pass

### ISSUE THREAD

**Issue by nemphys** [AOEpeople/Aoe_Scheduler#118]
This is a feature request and would allow eg. setting a lower priority to the longest running jobs, so that they don't delay all the rest when they get executed.

**Comment by fbrnc** [comment]
Hi @nemphys,

that's a great suggestion and should be easy to implement. I'll definitely take this into account for the next release.

Another related idea is to allow Aoe_Scheduler to execute all tasks a.) in an isolation process and b.) run those processes in parallel. 
But since this will probably be a major task I like to idea of sorting the jobs by priority and allowing to influence the order this way. 

In any case do you think the priority value should be more important than the scheduled time? So if task A was scheduled for 1:00 and has a lower priority and task B was scheduled at 1:05 with a higher priority. Which task do you think should be executed first?

**Comment by nemphys** [comment]
Thanx for taking the time to look into this.
Until you implement parallel processing (though this might pose other issues), I suppose it should be better to follow the current logic, ie. all tasks scheduled for eg. 1:00 should finish execution before the next cycle is started; that means that the scheduled time comes first, and priority follows.
Actually, I was thinking of a per-execution-cycle priority, not a global priority (again, the latter might introduce weird issues).
